lib/aligning_guidelines.js

```diff
@@ -52,11 +52,13 @@ function initAligningGuidelines(canvas) {
   }
 
   var verticalLines = [],
-      horizontalLines = [];
+      horizontalLines = [],
+      touchInProgress = false;
 
   canvas.on('mouse:down', function () {
     viewportTransform = canvas.viewportTransform;
     zoom = canvas.getZoom();
+    touchInProgress = true;
```

Comment:
I would be carefull calling this `touch` if is on mouse events and touch events.
Maybe we can find a name that is neutral between mouse and touch events.

Reply:
DONE